src/server.js

```diff
@@ -13,6 +13,8 @@ app.engine(".hbs", engine({ extname: ".hbs" }));
 app.set("view engine", ".hbs");
 app.set("views", path.resolve(__dirname, "./views"));
 
+app.use(express.static("public"));
```

Comment:
All static asset requests will be looked up in the `public` directory, located in the root of the project.

Reply:
I added a similar line below on line 40ish, can probably combine the two

Reply:
[Done here](https://github.com/monicaleep/member-fe-microsite/pull/17/commits/1e87ef1d7af6a2734d8cdc794a77a4c85e2e6c04)